events/wmt21.md

```diff
@@ -209,3 +209,47 @@ Thursday, November 11, 2021
 | 15:00 – 15:15 | [***Subword Mapping and Anchoring across Languages***](https://arxiv.org/pdf/2109.04556.pdf) <br>Giorgos Vernikos and Andrei Popescu-Belis |
 | 15:15 – 15:30 | [***Uncertainty-Aware Machine Translation Evaluation***](https://arxiv.org/pdf/2109.06352.pdf) <br>Taisiya Glushkova, Chrysoula Zerva, Ricardo Rei and André F. T. Martins |
 | 15:30 – 15:45 | [***Sometimes We Want Ungrammatical Translations***](https://aclanthology.org/2021.findings-emnlp.275.pdf) <br>Prasanna Parasarathi |
+
+## Results
+
+#### Full results of the shared tasks: [*Findings of the 2021 Conference on Machine Translation (WMT21)*](https://statmt.org/wmt21/pdf/2021.wmt-1.1.pdf)
```

Comment:
Hi dear @liashahnazaryan!

I've just realized that we have followed this approach when talking about conferences:

- When a talk refers to a published paper, its mention is formatted in italics and bold.  For instance, in [WMT21](https://machinetranslate.org/wmt21#day-1).
- When there is no published paper about it, its name is just bold, as in [MTM 2019](https://machinetranslate.org/mtm2019).

Thanks!

Reply:
hey, @cefoo!
I've made changes accordingly.